test/e2e/services.go

```diff
@@ -1,11 +1,15 @@
-// +build integration interactive
+//go:build integration || interactive || experimentalintegration
```

Comment:
I think here we need to keep the `// +build` syntax, since we are using go 1.16 (as mentioned in the docs [here](https://go.dev/doc/go1.16#:~:text=Note%20that%20%2F%2Fgo%3Abuild,introduced%20into%20Go%20programs%20yet.))

Reply:
from the [docs](https://go.dev/doc/go1.16#:~:text=Note%20that%20%2F%2Fgo%3Abuild,introduced%20into%20Go%20programs%20yet.): 
"Note that //go:build lines are not supported in Go 1.16 and should not be introduced into Go programs yet." 

As far as I understand looks like for 1.16 is recommended to use only `// + build` - I think we could for now follow this and when updating to `1.17` update the build tags to use the newest syntax
Could be a bit annoying with your setup - maybe you could install go 1.16 as well and configure to use it just with observatorium/api project?